python2/koans/about_tuples.py

```diff
@@ -7,14 +7,14 @@
 class AboutTuples(Koan):
     def test_creating_a_tuple(self):
         count_of_three = (1, 2, 5)
-        self.assertEqual(__, count_of_three[2])
+        self.assertEqual(5, count_of_three[2])
 
     def test_tuples_are_immutable_so_item_assignment_is_not_possible(self):
         count_of_three = (1, 2, 5)
         try:
             count_of_three[2] = "three"
         except TypeError as ex:
-            self.assertMatch(__, ex[0])
+            self.assertMatch("'tuple' object does not support item assignment", ex[0])
```

Comment:
Compare a list to a tuple. What do they have in common and what are their differences?

Reply:
The common is that they both have members that can be accessed using the `[]` operator, the difference is that a tuple cannot be modified while the list can.